test/ng/directive/ngRepeatSpec.js

```diff
@@ -1488,7 +1488,13 @@ describe('ngRepeat animations', function() {
     $rootScope.items = ['1','3'];
     $rootScope.$digest();
 
-    item = $animate.queue.shift();
+    while ($animate.queue.length) {
+      item = $animate.queue.shift();
+      if (item.event === 'leave') {
+        break;
+      }
+    }
```

Comment:
I am going to revert this code. The ngRepeat tests assume the animation queue is in a certain order. During refactoring of ngRepeat, the order was different in this test so I altered the test. My latest version of ngRepeat, c969ae23c48eab32989435d8ed4f9767e89c5ad4, passes the original version of this test.